virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -5,13 +5,15 @@ include::modules/virt-document-attributes.adoc[]
 
 toc::[]
 
-{VirtProductName} provides layer-2 networking capabilities that allow you to connect
-virtual machines to multiple networks. You can import virtual machines with
-existing workloads that depend on access to multiple interfaces. You can also
-configure a PXE network so that you can boot machines over the network.
+You can attach virtual machines to multiple networks using Linux bridges.
+You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
 
-To get started, a network administrator configures a bridge network attachment definition
-for a namespace in the web console or CLI. Users can then create a NIC to attach pods and virtual machines in that namespace to the bridge network.
+To attach a virtual machine to an additional network:
```

Comment:
This really resembles a procedure, though I understand why you wrote it this way. Maybe you could retain the overall structure but remove the numbered steps.

Reply:
It is a flow of required operations so you can't attach the VM to the network (2) if you haven't configured the bridge net-attach-def (1). I think it's better to be explicit so I've left the numbering for now but changed the sub-points as suggest. 
For comparison, here's a shot of it without any numbering:
![Screenshot from 2021-04-07 11-59-03](https://user-images.githubusercontent.com/17755748/113848701-07b49e00-9799-11eb-9f42-d7b6956965d4.png)